llvm/include/llvm/Support/PropertySetIO.h

```diff
@@ -123,6 +123,7 @@ class PropertySetRegistry {
   // Specific property category names used by tools.
   static constexpr char SYCL_SPECIALIZATION_CONSTANTS[] =
       "SYCL/specialization constants";
+  static constexpr char SYCL_DEVICELIB_REQ_MASK[] = "SYCL/devicelib req mask";
```

Comment:
```suggestion
```

Reply:
this should be moved to your new RT-LLVM shared header

Reply:
Done.